tests/test_utils_network.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+from collections.abc import AsyncGenerator
+
+import pytest
+import pytest_asyncio
+
+from imagedephi.utils.network import wait_for_port
+
+
+@pytest_asyncio.fixture
+async def server(unused_tcp_port: int) -> AsyncGenerator[asyncio.Server, None]:
+    def server_callback(reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
+        pass
+
+    server = await asyncio.start_server(server_callback, "127.0.0.1", unused_tcp_port)
+    async with server:
+        yield server
+        server.sockets[0]
+
+
+@pytest.mark.timeout(1)
+@pytest.mark.asyncio
+async def test_utils_network_wait_for_port(server: asyncio.Server) -> None:
+    server_port = server.sockets[0].getsockname()[1]
```

Comment:
Get `server_port` directly from `server`, instead of relying on the fact that `unused_tcp_port` is shared between the test case and the fixture (which is true, but makes the test dependent on the implementation of the fixture).